lib/cc/engine/analyzers/engine_config.rb

```diff
@@ -0,0 +1,49 @@
+module CC
+  module Engine
+    module Analyzers
+      class EngineConfig
+        attr_reader :config
+
+        def initialize(hash)
+          normalize(hash)
+        end
+
+        def language(language)
+          config.
+            fetch("languages", {}).
```

Comment:
I think paranoia is probably appropriate here.

That said, out of curiosity, what is the engine's behavior if no languages are specified?